tuxemon/npc.py

```diff
@@ -847,5 +847,87 @@ def alter_item_quantity(
 
         return success
 
+    def can_buy_item(self, item_slug: str, qty: int, unit_price: int) -> bool:
+        current_money = self.game_variables.get("money", None)
```

Comment:
This is picky, but `None` is the default argument for `get` so you don't need to specify it. 
Just do the following:
```
        current_money = self.game_variables.get("money")
```

I checked the rest of the repo and there are only 3 cases where we use .get(key, None) and 63 cases where we use .get(key), so this way is more consistent.